### PR TITLE
WP compatibility testing

### PR DESCRIPTION
**Pull request by akii09**
### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: .github/workflows/push-to-deploy.yml [hunk 1]
```diff
@@ -10,7 +10,8 @@ jobs:
     steps:
     - uses: actions/checkout@master
     - name: WordPress Plugin Deploy
-      uses: 10up/action-wordpress-plugin-deploy@master
+      uses: 10up/action-wordpress-plugin-deploy@stable
```
Comment (sushmak02) [inline .github/workflows/push-to-deploy.yml:13]: revert these changes

Comment (Nikschavan) [reply .github/workflows/push-to-deploy.yml:13]: Actually stable is the recommended branch for this action - https://github.com/10up/action-wordpress-plugin-deploy#deploy-on-pushing-a-new-tag

**Review by sushmak02 — APPROVED** [review]
